src/data/staking-products.json

```diff
@@ -636,6 +636,35 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Sensei Node go to link"
       }
+    },
+    {
+      "name": "Figment",
+      "imageName": "Figment",
+      "hue": 52,
+      "launchDate": "2021-12-31",
+      "url": "https://figment.io/",
+      "audits": [],
```

Comment:
@ivanszeftel As noted, this is from them not being publicly available
<img width="597" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/8e02ec69-acf1-4b01-b41f-deb830a48078">